src/Octoshift/Services/OctoLogger.cs

```diff
@@ -78,7 +80,7 @@ private string FormatMessage(string msg, string level)
         return $"[{timeFormat}] [{level}] {msg}\n";
     }
 
-    private string MaskSecrets(string msg)
+    private string Redact(string msg)
```

Comment:
nit: I like the old name better. To me redaction is deletion of something but masking is replacing the sensitive data with some mask characters.

Reply:
I prefer `Redact` now that it masks secrets and generic, hard-coded patterns. But I am happy to leave the name as it is rather than change things, since you don't feel the same 😊

Reply:
I don't feel strongly about it, will leave it up to you :)

Reply:
I'm going to go for `Redact` as I do think it's clearer - done in 062d0c2